models/drift_model.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"github.com/kamva/mgm/v3"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Drift struct {
+	mgm.DefaultModel `bson:",inline"`
+	Challenges       []primitive.ObjectID `bson:"challenges" json:"challenges" binding:"required"`
+	Prize            string               `bson:"prize" json:"prize" binding:"required"`
```

Comment:
I don't think we need a Prize field here. Prizes should be redeemable outside of a drift (via a User's points) so they shouldn't be bound to a specific Drift. The only reward from a drift is points, which are then used to "purchase" Prizes outside of a Drift.

Reply:
Hmm, we still have to hash this out in terms of our business plan, but I thought the idea was that at the end of any drift, a prize is awarded? I'm okay with either/or, but for now maybe just remove the `binding:"requred"` and keep it as an optional field and we can discuss this more in a meeting?

Reply:
discussed in the meeting. good as it is for now